guides/15_Background_Filter_Video_Processor.md

```diff
@@ -1,3 +1,8 @@
+**Note**
+For updated versions of background blur and background replacement, please use the `VideoFXProcessor`. Use the guide at [https://aws.github.io/amazon-chime-sdk-js/modules/backgroundfilter_videofx_processor.html](https://aws.github.io/amazon-chime-sdk-js/modules/backgroundfilter_videofx_processor.html)
```

Comment:
Please mention the Android support here.

Reply:
Asking my manager to confirm the change. WIll update after that.

Reply:
So we need approval from PM (i.e. Chari) for this. Since we clearly mentioned about not supporting Android devices on the new guide, I think it is ok to not mention it here.

Reply:
Discussed this offline!